android/app/src/main/res/values/dimens.xml

```diff
@@ -40,7 +42,7 @@
     <dimen name="heading_text_size">20sp</dimen>
     <dimen name="text_size_expanded_title">22sp</dimen>
     <dimen name="text_size_extra_large">18sp</dimen>
-    <dimen name="text_size_large">18sp</dimen>
+    <dimen name="text_size_large">16sp</dimen>
```

Comment:
@shreyasnbhat 
please undo this change. or i will send a pr separately if this is needed in your pr.
please suggest accordingly.